### PR TITLE
Removed PG connections and the refresh button from the import wizard.

### PR DESCRIPTION
**Pull request by aasimkhan30**
- Sending an error message when a PG database is selected for the wizard
- Removed refresh button #6611

This PR fixes #

### CODE REVIEW

File: extensions/import/src/wizard/pages/prosePreviewPage.ts [hunk 1]
```diff
@@ -57,8 +57,8 @@ export class ProsePreviewPage extends ImportPage {
 			},
 			{
 				component: this.table,
-				title: '',
-				actions: [this.refresh]
+				title: ''
+				//actions: [this.refresh]  Removing this button until it makes sense (Issue #6611)
```
Comment (aaomidi) [inline extensions/import/src/wizard/pages/prosePreviewPage.ts:61]: I wouldn't remove this yet, it's not hurting anyone being there and some users might be changing their CSV files and don't want to go through the wizard every time they do that.

Comment (aasimkhan30) [reply extensions/import/src/wizard/pages/prosePreviewPage.ts:61]: Okay, makes sense I'll leave it there for now.

File: extensions/import/src/wizard/flatFileWizard.ts [hunk 2]
```diff
@@ -39,7 +39,12 @@ export class FlatFileWizard {
 		let pages: Map<number, ImportPage> = new Map<number, ImportPage>();
 
 		let connectionId = (await azdata.connection.getCurrentConnection())?.connectionId ??
-			(await azdata.connection.openConnectionDialog())?.connectionId;
+			(await azdata.connection.openConnectionDialog(['MSSQL']))?.connectionId;
+
+		if ((await azdata.connection.getCurrentConnection()).providerId !== 'MSSQL') {
```
Comment (Charles-Gagnon) [inline extensions/import/src/wizard/flatFileWizard.ts:44]: If we already have a connection there's no reason to call this twice - just save the connection from the above line and use that instead.

File: extensions/import/src/wizard/flatFileWizard.ts [hunk 3]
```diff
@@ -38,14 +38,25 @@ export class FlatFileWizard {
...
+
+		if (!currentConnection) {
+			connectionId = (await azdata.connection.openConnectionDialog(['MSSQL'])).connectionId;
+			if (!connectionId) {
+				vscode.window.showErrorMessage(localize('import.needConnection', "Please connect to a server before using this wizard."));
+				return;
+			}
+		} else {
+			if (currentConnection.providerId !== 'MSSQL') {
+				vscode.window.showErrorMessage(localize('import.needSQLConnection', "The import extension currently support Microsoft SQL databases only. Please connect to a Microsoft SQL database before using this wizard."));
```
Comment (Charles-Gagnon) [inline extensions/import/src/wizard/flatFileWizard.ts:53]: `currently supports Microsoft SQL databases`

I'd also run this by a PM first to make sure this is the correct product naming to use.

Comment (kburtram) [reply extensions/import/src/wizard/flatFileWizard.ts:53]: Maybe something like `"SQL Server Import extension does not support this type of connection"` is fine.

File: extensions/import/src/wizard/flatFileWizard.ts [hunk 4]
```diff
@@ -38,14 +38,25 @@ export class FlatFileWizard {
...
+				return;
+			}
+			connectionId = currentConnection.providerId;
```
Comment (kburtram) [inline extensions/import/src/wizard/flatFileWizard.ts:56]: should this be `connectionId = currentConnection.connectionId ;` instead?

**Pull request merged**